decompressed/gui_file/www/docroot/modals/contentsharing-modal.lp

```diff
@@ -373,7 +373,7 @@ createUsbPartitionTable()
 function convertReceivedData(content)
 	if content["connecteddevices"] then
 		if usb_disk_number == 0 then
-			content["connecteddevices"] = format(T'<strong>%s</strong> connected devices', T"no")
+			content["connecteddevices"] = format(T'<strong>%d</strong> connected devices', T"0")
```

Comment:
%d e' solo per numeri "0" nel momento in cui e' tra virgolette e' visto come stringa non numero (tralasciando che poi T"0" mi sembra inutile)

Reply:
Grazie. Puoi per favore sistemarlo tu?
L'importante è non lasciarlo come ora che se non c'è nessun dispositivo connesso viene visualizzato
**_no connected devices_**
la mia proposta è
**_0 connected devices_**